packages/front-end/styles/global.scss

```diff
@@ -1374,7 +1396,7 @@ main.main.report {
 }
 
 .nav-link {
-  padding: 0.3rem 0.8rem;
+  padding: 0.3rem 1.5rem;
```

Comment:
Is this used in many places?  Does it look good in all of them?

Reply:
Yes